### ISSUE TITLE
validate doesn't report null `action` in `stories`

### ISSUE THREAD

**Issue by rgstephens** [RasaHQ/rasa#9113]
**Rasa version**: 2.2

**Issue**: `rasa data validate` does not report a null action in a story. `rasa train` will also generate a model without warning but there are dialogue prediction issues.

Example:

<pre>version: "2.0"
 stories:

*   story: 'Example store'
     steps:
*   slot_was_set:
*   user_status: member
*   intent: price_changes
*   action: </pre>

**Comment by hsm207** [comment]
This has been fixed in 3.0.3.

This story:

```
- story: sad path 2
  steps:
  - intent: greet
  - action: utter_greet
  - intent: mood_unhappy
  - action: utter_cheer_up
  - action: utter_did_that_help
  - intent: deny
  - action: 
 ```
Will give the following warning (from `rasa data validate`):

```
/opt/venv/lib/python3.8/site-packages/rasa/shared/utils/io.py:99: UserWarning: Issue found in 'data/stories.yml': 
Action name cannot be empty. This story step will be skipped:
{'action': None}
  More info at https://rasa.com/docs/rasa/stories
```

**Comment by m-vdb** [comment]
Closing as stale
